File: pyg_web_manager/src/main/java/com/itheima/pyg/controller/userCount/UserCountController.java
package com.itheima.pyg.controller.userCount;

import com.alibaba.dubbo.config.annotation.Reference;
import com.itheima.pyg.service.user.UserService;
import com.itheima.pyg.service.userCount.UserCount;
import com.sun.org.apache.regexp.internal.RE;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/userCount")
public class UserCountController {

    @Reference
    private UserCount userCount;


    @RequestMapping("/findUser")
    public long findUser() {

       /* System.out.println("用户人数"+user);*/
        return userCount.findUser();
    }

}

File: pyg_web_manager/src/main/webapp/js/controller/userCountController.js
//控制层
app.controller('userCountController', function ($scope, userService) {
//用户统计
    $scope.findUser = function () {

        userService.findUser().success(function (response) {
            $scope.num = response;
            alert("当前用户总人数为:"+num);
        })
    };
});